### PR TITLE
Merge dev stage

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 1]
```diff
@@ -9965,6 +10026,13 @@ span#HW_badge_cont:nth-child(3){
     .pagination>li>span {
         padding: 6px 6px;
     }
+    // header .hamburger_menu h4 {
+    //     white-space: nowrap;
+    //     overflow: hidden;
+    //     text-overflow: ellipsis;
+    //     min-height: 20px;
+    //     width: calc(100% - 30px);
+    // }
```
Comment (rohit-walkover) [inline apps/web-giddh/src/assets/css/style-2.scss:10035]: Remove commented code if not required

File: libs/utils/platform.ts [hunk 2]
```diff
@@ -81,37 +81,37 @@ export function download(filename, data, mimeType) {
...
+                            //alert("Unable to download");
+                            //console.error(err);
                         }
                     );
                 },
                 (err) => {
-                    alert("Unable to download");
-                    console.error(err);
+                    //alert("Unable to download");
+                    //console.error(err);
```
Comment (rohit-walkover) [inline libs/utils/platform.ts:114]: Remove commented code if not required

File: libs/utils/platform.ts [hunk 3]
```diff
@@ -81,37 +81,37 @@ export function download(filename, data, mimeType) {
...
+                                        //alert("Unable to download");
+                                        //console.error(err);
                                     };
                                 },
                                 (err) => {
                                     // failed
-                                    alert("Unable to download");
-                                    console.error(err);
+                                    //alert("Unable to download");
+                                    //console.error(err);
```
Comment (rohit-walkover) [inline libs/utils/platform.ts:102]: Remove commented code if not required

File: libs/utils/platform.ts [hunk 4]
```diff
@@ -81,37 +81,37 @@ export function download(filename, data, mimeType) {
...
                                                 // debugger;
-                                                console.error(err);
-                                                alert("Unable to download");
+                                                //console.error(err);
+                                                alert("No app available to open this type of file.");
                                             },
                                             success: () => {
                                                 // debugger;
-                                                console.log("success with opening the file");
+                                                //alert("success with opening the file");
```
Comment (rohit-walkover) [inline libs/utils/platform.ts:89]: Remove commented code if not required

File: libs/utils/platform.ts [hunk 5]
```diff
@@ -81,37 +81,37 @@ export function download(filename, data, mimeType) {
...
+                                    //alert("Unable to download");
+                                    //console.error(err);
                                 }
                             );
                         },
                         (err) => {
-                            alert("Unable to download");
-                            console.error(err);
+                            //alert("Unable to download");
+                            //console.error(err);
```
Comment (rohit-walkover) [inline libs/utils/platform.ts:108]: Remove commented code if not required

File: libs/utils/platform.ts (hunk 2 above)
Comment (shalinee-walkover) [reply libs/utils/platform.ts:114]: It will be reuse for mobile apk

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss:10035]: It will be reuse for mobile apk

**Pull request merged**